app/tests/__init__.py

```diff
@@ -0,0 +1,130 @@
+import logging
+from datetime import timedelta
+from pprint import pformat
+from urllib.parse import urlparse
+
+from django.test import TestCase
+
+from stac_api.utils import fromisoformat
+
+logger = logging.getLogger(__name__)
+
+
+def mock_request_from_response(factory, response):
+    '''Mock a request from a client response
+
+    This can be used to verify a client response against the manually serialized response data.
+    Some serializer require a request context in order to generate links.
+    '''
+    return factory.get(f'{response.request["PATH_INFO"]}?{response.request["QUERY_STRING"]}')
+
+
```

Comment:
I'd move the content in this file to a dedicated file, like `basetest.py` or `common.py` or sth like this...